HPXMLtoOpenStudio/resources/hpxml.rb

```diff
@@ -3382,7 +3387,9 @@ class CoolingSystem < BaseElement
              :cooling_efficiency_seer, :cooling_efficiency_eer, :cooling_efficiency_ceer, :cooling_efficiency_kw_per_ton,
              :cooling_shr, :third_party_certification, :clg_seed_id, :is_shared_system, :number_of_units_served,
              :shared_loop_watts, :shared_loop_motor_efficiency, :fan_coil_watts, :airflow_defect_ratio,
-             :fan_watts_per_cfm, :charge_defect_ratio, :cooling_airflow_cfm, :location, :primary_system]
+             :fan_watts_per_cfm, :charge_defect_ratio, :cooling_airflow_cfm, :location, :primary_system, :attached_heating_system_fuel,
+             :attached_heating_system_capacity, :attached_heating_system_efficiency, :attached_heating_system_fraction_heat_load_served,
```

Comment:
@shorowit Currently I implemented it in the way that cooling system instance carries all the attached heating information, including autosized heating capacity, airflow, etc. This is consistent with hpxml schema cooling system description. An alternative can be to create a heating system instance in the workflow to carry the information related to heating. That might simplify some changes in the current PR that enables cooling system to have heating features. But haven't thought it through that whether it will mess up with other heating systems or cause other issues.